mffpy/xml_files.py

```diff
@@ -495,6 +579,20 @@ def content(cls, epochs: List[Epoch]) -> dict:  # type: ignore
             ]
         }
 
+    def get_content(self):
```

Comment:
Seeing all the methods of this `get_content`, `get_serialized_content` pattern, it may be good to spend a bit of time looking at them to see if we can condense them down, if not now, in an upcoming improvement to the feature.